docs/en/01_Tutorials/02_Extending_A_Basic_Site.md

```diff
@@ -462,16 +462,19 @@ Nothing here should be new. The *StaffPage* page type is more interesting though
 	:::php
 	<?php
 	class StaffPage extends Page {
-		private static $db = array(
-		);
+
 		private static $has_one = array(
-			'Photo' => 'Image'
+			'Photo' => 'File'
 		);
 		
 		public function getCMSFields() {
 			$fields = parent::getCMSFields();
 			
-			$fields->addFieldToTab("Root.Images", new UploadField('Photo'));
+			$fields->addFieldToTab(
+				"Root.Images",
+				UploadField::create('Photo')
+					->setAllowedFileCategories('image')
```

Comment:
Irritating I'm hitting the docs before the code :). Why a string, and not a const?